[2018-09-03 11:32:24,933] {cli.py:374} INFO - Running on host MacBook-Pro-de-MANGANNEAU.local
[2018-09-03 11:32:24,954] {models.py:1197} INFO - Dependencies all met for <TaskInstance: workshop_airflow_exo_1.send_mail 2018-09-03 09:32:01.427773 [queued]>
[2018-09-03 11:32:24,963] {models.py:1197} INFO - Dependencies all met for <TaskInstance: workshop_airflow_exo_1.send_mail 2018-09-03 09:32:01.427773 [queued]>
[2018-09-03 11:32:24,963] {models.py:1407} INFO - 
--------------------------------------------------------------------------------
Starting attempt 1 of 1
--------------------------------------------------------------------------------

[2018-09-03 11:32:24,969] {models.py:1428} INFO - Executing <Task(EmailOperator): send_mail> on 2018-09-03 09:32:01.427773
[2018-09-03 11:32:24,969] {base_task_runner.py:115} INFO - Running: ['bash', '-c', 'airflow run workshop_airflow_exo_1 send_mail 2018-09-03T09:32:01.427773 --job_id 12 --raw -sd DAGS_FOLDER/exo1.py']
[2018-09-03 11:32:25,564] {base_task_runner.py:98} INFO - Subtask: [2018-09-03 11:32:25,563] {__init__.py:45} INFO - Using executor SequentialExecutor
[2018-09-03 11:32:25,609] {base_task_runner.py:98} INFO - Subtask: [2018-09-03 11:32:25,608] {models.py:189} INFO - Filling up the DagBag from /Users/magus/PycharmProjects/workshop_airflow/dags/exo1.py
[2018-09-03 11:32:25,701] {cli.py:374} INFO - Running on host MacBook-Pro-de-MANGANNEAU.local
[2018-09-03 11:32:25,718] {base_task_runner.py:98} INFO - Subtask: [2018-09-03 11:32:25,718] {configuration.py:206} WARNING - section/key [smtp/smtp_password] not found in config
[2018-09-03 11:32:26,188] {base_task_runner.py:98} INFO - Subtask: [2018-09-03 11:32:26,188] {email.py:112} INFO - Sent an alert email to ['guy-stephane.manganneau@openvalue.fr']
[2018-09-03 11:32:26,288] {models.py:1595} ERROR - (550, b'Unauthenticated senders not allowed', 'airflow@example.com')
Traceback (most recent call last):
  File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/models.py", line 1493, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/operators/email_operator.py", line 64, in execute
    send_email(self.to, self.subject, self.html_content, files=self.files, cc=self.cc, bcc=self.bcc, mime_subtype=self.mime_subtype)
  File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/utils/email.py", line 44, in send_email
    return backend(to, subject, html_content, files=files, dryrun=dryrun, cc=cc, bcc=bcc, mime_subtype=mime_subtype)
  File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/utils/email.py", line 87, in send_email_smtp
    send_MIME_email(SMTP_MAIL_FROM, recipients, msg, dryrun)
  File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/utils/email.py", line 113, in send_MIME_email
    s.sendmail(e_from, e_to, mime_msg.as_string())
  File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/smtplib.py", line 867, in sendmail
    raise SMTPSenderRefused(code, resp, from_addr)
smtplib.SMTPSenderRefused: (550, b'Unauthenticated senders not allowed', 'airflow@example.com')
[2018-09-03 11:32:26,294] {models.py:1624} INFO - Marking task as FAILED.
[2018-09-03 11:32:26,304] {models.py:1644} ERROR - (550, b'Unauthenticated senders not allowed', 'airflow@example.com')
[2018-09-03 11:32:26,307] {base_task_runner.py:98} INFO - Subtask: Traceback (most recent call last):
[2018-09-03 11:32:26,307] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/bin/airflow", line 27, in <module>
[2018-09-03 11:32:26,307] {base_task_runner.py:98} INFO - Subtask:     args.func(args)
[2018-09-03 11:32:26,307] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/bin/cli.py", line 392, in run
[2018-09-03 11:32:26,307] {base_task_runner.py:98} INFO - Subtask:     pool=args.pool,
[2018-09-03 11:32:26,308] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/utils/db.py", line 50, in wrapper
[2018-09-03 11:32:26,308] {base_task_runner.py:98} INFO - Subtask:     result = func(*args, **kwargs)
[2018-09-03 11:32:26,308] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/models.py", line 1493, in _run_raw_task
[2018-09-03 11:32:26,308] {base_task_runner.py:98} INFO - Subtask:     result = task_copy.execute(context=context)
[2018-09-03 11:32:26,308] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/operators/email_operator.py", line 64, in execute
[2018-09-03 11:32:26,308] {base_task_runner.py:98} INFO - Subtask:     send_email(self.to, self.subject, self.html_content, files=self.files, cc=self.cc, bcc=self.bcc, mime_subtype=self.mime_subtype)
[2018-09-03 11:32:26,308] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/utils/email.py", line 44, in send_email
[2018-09-03 11:32:26,309] {base_task_runner.py:98} INFO - Subtask:     return backend(to, subject, html_content, files=files, dryrun=dryrun, cc=cc, bcc=bcc, mime_subtype=mime_subtype)
[2018-09-03 11:32:26,309] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/utils/email.py", line 87, in send_email_smtp
[2018-09-03 11:32:26,309] {base_task_runner.py:98} INFO - Subtask:     send_MIME_email(SMTP_MAIL_FROM, recipients, msg, dryrun)
[2018-09-03 11:32:26,309] {base_task_runner.py:98} INFO - Subtask:   File "/Users/magus/PycharmProjects/workshop_airflow/workshop_env/lib/python3.6/site-packages/airflow/utils/email.py", line 113, in send_MIME_email
[2018-09-03 11:32:26,309] {base_task_runner.py:98} INFO - Subtask:     s.sendmail(e_from, e_to, mime_msg.as_string())
[2018-09-03 11:32:26,309] {base_task_runner.py:98} INFO - Subtask:   File "/Library/Frameworks/Python.framework/Versions/3.6/lib/python3.6/smtplib.py", line 867, in sendmail
[2018-09-03 11:32:26,393] {base_task_runner.py:98} INFO - Subtask:     raise SMTPSenderRefused(code, resp, from_addr)
[2018-09-03 11:32:26,394] {base_task_runner.py:98} INFO - Subtask: smtplib.SMTPSenderRefused: (550, b'Unauthenticated senders not allowed', 'airflow@example.com')
